pom.xml

```diff
@@ -71,7 +71,7 @@
         <maven.findbugs.plugin.version>3.0.1</maven.findbugs.plugin.version>
         <maven.sonar.plugin.version>2.6</maven.sonar.plugin.version>
         <maven.jacoco.plugin.version>0.7.4.201502262128</maven.jacoco.plugin.version>
-        <maven.failsafe.plugin.version>2.17</maven.failsafe.plugin.version>
+        <maven.failsafe.plugin.version>2.19</maven.failsafe.plugin.version>
```

Comment:
the older failfast plugin has a bug - it places the classes/resources on a classpath and not the generated JAR.